doc/source/dev/missing-bits.rst

```diff
@@ -104,8 +104,9 @@ Finally, hide private attributes if any::
 Test functions from `numpy.testing`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 In new code, don't use `assert_almost_equal`, `assert_approx_equal` or
-`assert_array_almost_equal`. This is from the docstrings of these
-functions::
+`assert_array_almost_equal`. In addition, do not use `assert_warns` with a
+`match` parameter, as this ignores that parameter. This is from the docstrings of
+these functions::
```

Comment:
minor, but I would put this new sentence after the quoted sentence, in a paragraph of its own.

Reply:
Actually, I thought this would go in the next section, "Testing that expected exceptions and warnings are raised". What do you think about that?

Reply:
Yep, that makes sense actually.

Reply:
Just adjusted this - I haven't worked on the docs much so hopefully my formatting was correct.